### PR TITLE
Story/vspc 95

### PR DESCRIPTION
**Pull request by Sagar-k93**
Changes to make the public slides page. The contents of the slide are scroll-able but the slide control buttons are to be static.

### CODE REVIEW

File: vspace/src/main/java/edu/asu/diging/vspace/core/model/ISpaceLink.java [hunk 1]
```diff
@@ -2,12 +2,11 @@
 
 public interface ISpaceLink extends ILink<ISpace> {
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/core/model/ISpaceLink.java:3]: Indentation and line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/core/model/IPrefix.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
-package edu.asu.diging.vspace.core.model;
-
-public interface IPrefix {
-
-    public static String SPACE_PREFIX = "SPA";
-    public static String MODULE_PREFIX = "MOD";
-}
+package edu.asu.diging.vspace.core.model;
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/core/model/IPrefix.java:1]: Line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/core/model/IdPrefix.java [hunk 3]
```diff
@@ -1,17 +1,17 @@
-package edu.asu.diging.vspace.core.model;
-
-public enum IdPrefix {
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/core/model/IdPrefix.java:3]: Line ending changes.

File: vspace/src/main/webapp/WEB-INF/views/login.html [hunk 4]
```diff
@@ -1,60 +1,60 @@
-<html layout:decorate="~{layouts/main}">
```
Comment (Sagar-k93) [inline vspace/src/main/webapp/WEB-INF/views/login.html:1]: Line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Space.java [hunk 5]
```diff
@@ -30,7 +30,9 @@
 
     @Id
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Space.java:31]: Spacing changes in this file.

File: vspace/src/main/java/edu/asu/diging/vspace/web/HomeController.java [hunk 6]
```diff
@@ -1,47 +1,47 @@
-package edu.asu.diging.vspace.web;
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/web/HomeController.java:1]: Spaces and line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddSpaceController.java [hunk 7]
```diff
@@ -1,89 +1,89 @@
-package edu.asu.diging.vspace.web.staff;
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddSpaceController.java:1]: Line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java [hunk 8]
```diff
@@ -1,68 +1,68 @@
-package edu.asu.diging.vspace.web.staff;
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java:1]: Line ending changes.

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/SequenceController.java [hunk 9]
```diff
@@ -1,48 +1,48 @@
-package edu.asu.diging.vspace.web.staff;
```
Comment (Sagar-k93) [inline vspace/src/main/java/edu/asu/diging/vspace/web/staff/SequenceController.java:1]: Line ending changes.

File: Docker/.gitignore [hunk 10]
```diff
@@ -1 +0,0 @@
-/.DS_Store
```
Comment (jdamerow) [inline Docker/.gitignore:1]: should not be deleted

File: .project [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+<projectDescription>
+	<name>virtual-spaces-2.0</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```
Comment (jdamerow) [inline .project:11]: let's not commit this file

File: vspace/.classpath [hunk 12]
```diff
@@ -1,50 +1,50 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jdamerow) [inline vspace/.classpath:1]: this file seems to have the wrong line endings

**Review by jdamerow — CHANGES_REQUESTED** [review]
Please reread pull request guidelines before I look at this again.

**Pull request closed without merging**